lib/barbeque/message/job_execution.rb

```diff
@@ -13,7 +13,11 @@ def assign_body(message_body)
         super
         @application = message_body['Application']
         @job  = message_body['Job']
-        @body = message_body['Message']
+        @body = if message_body['Message'].is_a?(String)
```

Comment:
I think garage_client will send with `Content-Body: application/json`. Is there really such a use case?